library/alloc/src/string.rs

```diff
@@ -403,6 +403,9 @@ impl String {
     /// s.push('a');
     /// ```
     #[inline]
+    #[doc(alias = "alloc")]
+    #[doc(alias = "calloc")]
```

Comment:
This is definitely not `calloc` here since `calloc` fills the content with a given value.